### PR TITLE
instead of port http server receives addr variable, minor fixes

### PR DESCRIPTION
**Pull request by axamon**

### CODE REVIEW

File: go-echo/api/bundle.go [hunk 1]
```diff
@@ -6,9 +6,21 @@ import (
...
+type Problem struct {
+	Title string
+	Status int32
+	Detail string
+}
+
+type Timestamps struct {
+	Ts time.Time
+}
+
```
Comment (ioggstream) [inline go-echo/api/bundle.go]: Those schemas should already be generated. If not please file an issue.

File: go-chi/main.go [hunk 2]
```diff
@@ -21,9 +21,9 @@ func main() {
 
 	s := &http.Server{
 		Handler: h,
-		Addr:    fmt.Sprintf("0.0.0.0:%d", *port),
+		Addr:   *addr,
 	}
 
 	// And we serve HTTP until the world ends.
-	log.Fatal(s.ListenAndServe())
+	log.Fatal(s.ListenAndServe ())
```
Comment (ioggstream) [inline go-chi/main.go]: Can you use `gofmt` to reformat code before commit?

File: go-chi/main.go [hunk 3]
```diff
@@ -21,9 +21,9 @@ func main() {
 
 	s := &http.Server{
 		Handler: h,
-		Addr:    fmt.Sprintf("0.0.0.0:%d", *port),
+		Addr:   *addr,
```
Comment (ioggstream) [inline go-chi/main.go]: not sure *addr is defined here.

File: go-echo/main.go [hunk 4]
```diff
@@ -50,5 +50,5 @@ func main() {
 	api.RegisterHandlers(e, petStore)
 
 	// And we serve HTTP until the world ends.
-	e.Logger.Fatal(e.Start(fmt.Sprintf("0.0.0.0:%d", *port)))
+	e.Logger.Fatal(e.Start(*addr)))
```
Comment (ioggstream) [inline go-echo/main.go]: Where did you define addr?

File: go-echo/api/bundle.go (hunk 1 above)
Comment (axamon) [reply go-echo/api/bundle.go]: opened issue missing types #7

File: go-chi/main.go (hunk 2 above)
Comment (axamon) [reply go-chi/main.go]: fixed nasty typo

File: go-chi/main.go (hunk 3 above)
Comment (axamon) [reply go-chi/main.go]: fixed changing port variable to addr

File: go-echo/main.go (hunk 4 above)
Comment (axamon) [reply go-echo/main.go]: fixed changing port variable to addr

File: go-echo/api/bundle.go (hunk 1 above)
Comment (axamon) [reply go-echo/api/bundle.go]: I removed the added types from the code and now the tests pass. Maybe issue can be closed,

**Comment by axamon** [comment]
ok all tests pass now

File: go-chi/go.mod [hunk 5]
```diff
@@ -5,6 +5,9 @@ go 1.12
 require (
 	github.com/deepmap/oapi-codegen v1.3.0
 	github.com/getkin/kin-openapi v0.2.0
+	github.com/go-chi/chi v4.0.2+incompatible
+	github.com/go-chi/cors v1.0.0
 	github.com/labstack/echo/v4 v4.1.11
+	github.com/stretchr/testify v1.4.0
 	gopkg.in/yaml.v2 v2.2.5
```
Comment (ioggstream) [inline go-chi/go.mod]: go.sum should probably be generated. for now don't change it and I should add it to the gitignore.

File: go-chi/main.go [hunk 6]
```diff
@@ -2,15 +2,14 @@ package main
...
-	"fmt"
 	"log"
 	"net/http"
 
 	api "github.com/ioggstream/simple/api"
 )
 
 func main() {
-	var port = flag.Int("port", 8080, "Port for test HTTP server")
+	var addr = flag.String("addr", "127.0.0.1:8080", "Address for test HTTP server")
```
Comment (ioggstream) [inline go-chi/main.go]: Ok!

File: go-echo/api/bundle.go [hunk 7]
```diff
@@ -6,9 +6,19 @@ import (
...
+// type Problem struct {
+// 	Title string
+// 	Status int32
+// 	Detail string
+// }
+
+// type Timestamps struct {
+// 	Ts time.Time
+// }
+
```
Comment (ioggstream) [inline go-echo/api/bundle.go]: yagni :)

File: go-chi/api/bundle.go [hunk 8]
```diff
@@ -12,6 +12,12 @@ import (
 	"time"
 )
 
+// type Problem struct {
```
[the remark this answers is not in the document]
Comment (axamon) [reply go-chi/api/bundle.go]: removed

File: go-echo/api/bundle.go (hunk 7 above)
Comment (axamon) [reply go-echo/api/bundle.go]: removed

File: go-chi/go.mod (hunk 5 above)
Comment (axamon) [reply go-chi/go.mod]: I believe go.mod should stay.
Should the code become a package of its own the go,mod should serve its purpose,

Comment (ioggstream) [reply go-chi/go.mod]: Ok, let's remove the vendor dir :D

Comment (axamon) [reply go-chi/go.mod]: vendor dir removed

**Comment by ioggstream** [comment]
Closed with #8 removing some unuseful files an with minor fixes. Thanks @axamon for the great work.

**Pull request closed without merging**